features/step_definitions/user_steps.rb

```diff
@@ -1,8 +1,30 @@
-Given /I am an existing user with email "(.*)" and password "(.*)" and age "(.*)" and height "(.*)" and weight "(.*)" and time "(.*)"/ do |email, password, age, height, weight, time|
-	User.create(:email => email, :password => password, :age => age, :height => height, :weight => weight, :time => time)
-	user = User.find_by_email('john.apple@gmail.com')
-	# @current_user = user
-	# sign_in_as user
+# Given /I am an existing user with email "(.*)"/ do |email|
```

Comment:
This looks great! I'm glad that we're stubbing out the tests the right way, this is going to make testing with cucumber far easier and efficient. Good job :)